action.yaml

```diff
@@ -66,7 +66,7 @@ runs:
         -H "Accept: application/vnd.heroku+json; version=3" \
         -H "Authorization: Bearer ${{ inputs.api-key }}"
     - name: Create Build
-      if: ${{ github.event_name == 'pull_request' && github.event.action == 'synchronize' }}
+      if: ${{ github.event_name == 'pull_request' && (github.event.action == 'opened' || github.event.action == 'synchronize') }}
```

Comment:
This shouldn't change. When a review app is created, it automatically has a build process going on. So from the `Create App` step we can use the data returned to find the output stream URL and curl that URL like it's done in `Create Build` step.